jobs/cronjob-ldap-group-sync.yml

```diff
@@ -7,44 +7,64 @@ metadata:
     description: "Scheduled Task to Perform LDAP Group Synchronization"
     iconClass: "icon-shadowman"
     tags: "management,cronjob,ldap,group,sync"
+labels:
+  template: "cronjob-ldap-group-sync"
+
 objects:
 - kind: ConfigMap
   apiVersion: v1
   metadata:
     name: ldap-config
+    namespace: "${NAMESPACE}"
     labels:
       template: "cronjob-ldap-group-sync"
   data:
-    ldap-sync.yml: |
-      kind: LDAPSyncConfig
-      apiVersion: v1
-      url: ${LDAP_URL}
-      insecure: true
-      bindDN: ${LDAP_BIND_DN}
-      bindPassword: ${LDAP_BIND_PASSWORD}
+    ldap-group-sync.yaml: |
+      kind: "LDAPSyncConfig"
+      apiVersion: "v1"
+      url: "${LDAP_URL}"
+      insecure: false
+      bindDN: "${LDAP_BIND_DN}"
+      bindPassword:
+        file: "/etc/secrets/bind_password"
+      ca: "/etc/config/ldap-ca.crt"
```

Comment:
See comment below re: `ldap` v.s. `ldaps`

Reply:
Fixed with LDAP_CA_CERT_PATH.

Nitpick... We are technically talking ldap (without TLS) vs. two ways that ldap implements TLS with ldap+starttls and also ldaps. So it isn't really `ldap` vs. `ldaps`.